lib/AST/Module.cpp

```diff
@@ -2425,6 +2425,19 @@ bool ModuleDecl::isStdlibModule() const {
   return !getParent() && getName() == getASTContext().StdlibModuleName;
 }
 
+bool ModuleDecl::isSomeStdlibModule() const {
+  return !getParent && (
```

Comment:
This is a fair number of comparisons, I wonder if it should be cached on `ModuleDecl`?